BrickController2/BrickController2.Android/MainActivity.cs

```diff
@@ -20,15 +23,75 @@ namespace BrickController2.Droid
             ConfigChanges.UiMode | 
             ConfigChanges.ScreenLayout | 
             ConfigChanges.SmallestScreenSize)]
-    public class MainActivity : MauiAppCompatActivity
+    public class MainActivity : MauiAppCompatActivity, InputManager.IInputDeviceListener
     {
         private readonly GameControllerService _gameControllerService;
+        private InputManager? _inputManager;
 
         public MainActivity()
         {
             _gameControllerService = IPlatformApplication.Current!.Services.GetRequiredService<GameControllerService>()!;
         }
 
+        protected override void OnCreate(Bundle? savedInstanceState)
+        {
+            base.OnCreate(savedInstanceState);
+
+            _inputManager = (InputManager?)GetSystemService(Context.InputService);
+
+            // JK: either register InputDeviceListener on Create/Destroy or Resume/Pause of MainActivity
+            // current decision: on Create/Destroy
+            _gameControllerService?.MainActivityOnCreate();
+            _inputManager?.RegisterInputDeviceListener(this, null);
+        }
+
+        protected override void OnDestroy()
+        {
+            // JK: OnDestroy was never called...
```

Comment:
What does this comment mean? If I apply Back gesture on the main page, I get this `OnDestry` called.

Reply:
OK? I've tested that on my Android smartphone and never reached a breakpoint inside "OnDestroy"...?!
Do you think it's okay to register/unregister the InputDeviceListener in OnCreate/OnDestroy?
Then I could remove all the OnPause/OnResume stuff?

Reply:
I hope so 😄

Reply:
I removed unused MainActivity stuff
https://github.com/vicocz/brickcontroller2/pull/75/commits/7a0d7a49ecdeef0e9aeccbefe61a99f870bbb670